pkg/kubectl/cmd/testdata/edit/testcase-not-update-annotation/2.request

```diff
@@ -1,7 +1,14 @@
 {
-  "metadata": {
-    "labels": {
-      "new-label": "new-value"
-    }
-  }
+	"metadata": {
+		"labels": {
+			"new-label": "new-value"
+		}
+	},
+	"spec": {
+		"$setElementOrder/ports": [
```

Comment:
@liggitt Which of the following does `no changes` mean?
- no order change
- no order change and no content change

This is important for implementation.